packages/connector-github/src/index.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  ConnectorMetadata,
+  GetAccessToken,
+  GetAuthorizationUri,
+  ValidateConfig,
+  GetUserInfo,
+  ConnectorError,
+  ConnectorErrorCodes,
+  SocialConnector,
+  GetConnectorConfig,
+} from '@logto/connector-types';
+import { assert } from '@silverhand/essentials';
+import got, { RequestError as GotRequestError } from 'got';
+import { stringify } from 'query-string';
+
+import {
+  authorizationEndpoint,
+  accessTokenEndpoint,
+  scope,
+  userInfoEndpoint,
+  githubConfigGuard,
+  GithubConfig,
+  defaultMetadata,
+  defaultTimeout,
+} from './constant';
+
+export class GithubConnector implements SocialConnector {
+  public metadata: ConnectorMetadata = defaultMetadata;
+
+  public readonly getConfig: GetConnectorConfig<GithubConfig>;
+
+  constructor(getConnectorConfig: GetConnectorConfig<GithubConfig>) {
+    this.getConfig = getConnectorConfig;
+  }
+
+  public validateConfig: ValidateConfig = async (config: unknown) => {
+    const result = githubConfigGuard.safeParse(config);
+
+    if (!result.success) {
+      throw new ConnectorError(ConnectorErrorCodes.InvalidConfig, result.error.message);
+    }
+  };
+
+  public getAuthorizationUri: GetAuthorizationUri = async (redirectUri, state) => {
+    const config = await this.getConfig(this.metadata.id);
+
+    return `${authorizationEndpoint}?${stringify({
+      client_id: config.clientId,
+      redirect_uri: redirectUri,
+      state,
+      scope, // Only support fixed scope for v1.
+    })}`;
```

Comment:
seems that it's unnecessary to use `query-string.stringify` and even `query-string`.

e.g.
![image](https://user-images.githubusercontent.com/10594507/165446164-91597dd3-d48d-4e35-803e-28bd1025b85e.png)

You can refactor it in another PR.